app/templates/_package.json

```
@@ -39,12 +55,27 @@
     "gulp-sourcemaps": "^2.2.0",
     "gulp-uglify": "^3.0.1",
     "gulp-useref": "^3.0.0",
-    "main-bower-files": "^2.5.0",
     "mocha": "^5.2.0",
     "run-sequence": "^2.2.1",
-    "wiredep": "^4.0.0",
     "yargs": "12.0.2"
-  },
+  }, 
+  "scripts": {
+    <% if (includeModernizr) { %>
+    "build:modernizr": "./node_modules/modernizr/bin/modernizr -c modernizr.json -d app/scripts/modernizr.js",
```

I suggest renaming this script to `modernizr` instead because it's not a variation of the `build` script, like `serve:dist` is a variation of `serve`. (I won't include this in the suggestion below because it has to be edited in multiple places.)

Also, within npm scripts you don't have to worry about targeting local executables, that behavior is built-in:

```suggestion
    "build:modernizr": "modernizr -c modernizr.json -d app/scripts/modernizr.js",
```